XLForm/XL/Cell/XLFormSelectorCell.m

```diff
@@ -32,13 +32,13 @@
 @interface XLFormSelectorCell() <UIActionSheetDelegate, UIPickerViewDelegate, UIPickerViewDataSource, UIPopoverControllerDelegate>
 
 @property (nonatomic) UIPickerView * pickerView;
-@property (nonatomic) UIPopoverController *popoverController;
 
 @end
 
 
 @implementation XLFormSelectorCell
 
+@synthesize popoverController;
```

Comment:
@jkereako  I know XLFormSelectorCell has `popoverController ` property that is part of `XLFormRowDescriptorPopoverViewController` protocol but XLFormRowDescriptorPopoverViewController is not intended to be used here.  Please restore the previous state adding back popoverController property and deleting popoverController synthesize.